src/App.scss

```diff
@@ -13,3 +13,67 @@ button.header-nav-panel-toggle{
   height: 38px;
   width: 82px;
 }
+h1{
+  font-family: 'Merriweather', serif;
+  font-size: 40px;
+  font-weight: bold;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.25;
+  letter-spacing: normal;
+  color: $dark-grey;
+}
+h2 {
+  font-family: "Open Sans";
+  font-size: 32px;
+  font-weight: 600;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.25;
+  letter-spacing: normal;
+  color: $dark-grey;
+}
+h2.subtitle {
+  font-size: 20px;
+  line-height: 1.5;
+}
+h3 {
+  font-family: "Open Sans";
+  font-size: 28px;
+  font-weight: 600;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.25;
+  letter-spacing: normal;
+  color: $dark-grey;
+}
+h4 {
+  font-family: "Open Sans";
+  font-size: 24px;
+  font-weight: 600;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.5;
+  letter-spacing: normal;
+  color: $dark-grey;
+}
+h5{
+  font-family: "Open Sans";
+  font-size: 20px;
+  font-weight: 600;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.5;
+  letter-spacing: normal;
+  color: $dark-grey;
+}
+p {
```

Comment:
It seems like duplication that `font-style`, `font-stretch`, and `letter-spacing` are all set to normal for ever item. If normal is the default setting for these (i.e. if nothing changes when you remove them), we can't exclude those settings. If they are needed, it would nice to wrap them in a mixin called base-font or something. See https://sass-lang.com/guide#topic-7-SCSS for mixin docs.